cmd/arc/services/metamorph.go

```diff
@@ -4,6 +4,10 @@ import (
 	"context"
 	"fmt"
 	"github.com/bitcoin-sv/arc/internal/cache"
+	grpcopts "github.com/bitcoin-sv/arc/internal/grpc_opts"
+	natscore "github.com/bitcoin-sv/arc/internal/message_queue/nats/client/nats_core"
+	natsjetstream "github.com/bitcoin-sv/arc/internal/message_queue/nats/client/nats_jetstream"
+	natsconnection "github.com/bitcoin-sv/arc/internal/message_queue/nats/nats_connection"
```

Comment:
No aliases needed here, as this rule has been excluded

Reply:
Removed